docs/development/translations.rst

```diff
@@ -0,0 +1,408 @@
+   :align: center
+   :figwidth: 80%
+   :width: 100%
+
+   Glossary sidebar in Weblate. Click on the image to see it full-size.
+
+If you find that a *source string* contains terms from the
+:doc:`SecureDrop glossary <../glossary>` or the `EFF Surveillance
+Self-Defense glossary`_, but the glossary sidebar says ``No related strings found in the glossary.``,
+we'd really appreciate it if you could add those terms to
+the glossary of the language you're working with.
+
+.. _weblate_glossary:
+
+Glossary
+--------
+
+Weblate contains an internal glossary for each language, to which
+we can add suggested translations. Learn more about
+:ref:`using language glossaries on Weblate <how_to_use_language_glossaries_on_weblate>`.
+
+.. figure:: ../images/weblate/glossary-list.png
+   :align: center
+   :figwidth: 80%
+   :width: 100%
+
+   A language glossary in Weblate. Click on the image to see it full-size.
+
+If a term is missing from the glossary for the language
+you're translating into, you can refer to the following
+technical glossaries for additional context. Then you can contribute
+to improving your own language glossary on Weblate by suggesting
+a translation yourself!
+
+- The :doc:`SecureDrop glossary <../glossary>` explains terms specific
+  to SecureDrop.
+- The `EFF Surveillance Self-Defense glossary`_ explains many general
+  security concepts.
+
+Additionally, here is a list of terms that are specific to the usage of Weblate for SecureDrop.
+
+.. _weblate_glossary_reviewer:
+
+Reviewer
+^^^^^^^^
+
+Reviewers are people who are trusted to review and accept new translations into SecureDrop.
+
+.. _weblate_glossary_source_string:
+
+Source string
+^^^^^^^^^^^^^
+
+On Weblate, the phrases being translated are called *source strings*. (No relation with the terms *source* and *journalist* in SecureDrop.)
+
+Source strings are English phrases and are automatically extracted from SecureDrop's code. Because of that, they can only be modified by developers outside Weblate. Learn :ref:`how to suggest changes to a source string <how_to_suggest_changes_to_a_source_string>`.
+
+.. _`Weblate`: https://weblate.org/
+.. _`SecureDrop Weblate instance`: https://weblate.securedrop.org/
+.. _`Weblate registration page`: https://weblate.securedrop.org/accounts/register/
+.. _`Weblate dashboard`: https://weblate.securedrop.org/
+.. _`translation category of the SecureDrop forum`: https://forum.securedrop.org/c/translations
+.. _`SecureDrop instant messaging channel`: https://gitter.im/freedomofpress/securedrop
+.. _`Weblate documentation`: https://docs.weblate.org/
+.. _`EFF Surveillance Self-Defense glossary`: https://ssd.eff.org/en/glossary/
+
+
+.. |Weblate dashboard screenshot| image::
+   ../images/weblate/dashboard.png
+   :width: 100%
+.. |Waiting for review screenshot| image::
+   ../images/weblate/waiting-for-review.png
+   :width: 100%
+.. |Approved screenshot| image::
+   ../images/weblate/approved.png
+   :width: 100%
```

Comment:
These three images are not used anywhere at this point because:

- the sections on reviewing and approving strings are now shorter and don't really call for an image IMHO
- it seems to me that the first Weblate screenshot that is shown is a good preview of Weblate and adding one specifically of the Dashboard is not really useful

What do you think? If we'll remove images, I suppose I should make sure to _untrack_ them from Git LFS and probably update something around [the script that generates them](https://github.com/freedomofpress/securedrop/blob/1.8.0/Makefile#L304-L313)? (I didn't find a list of files ot generate, and couldn't find where the list would be generated either, so I'm not sure about that last part.)

Reply:
I agree these screenshots can be removed. You don't need to do anything special vis-a-vis LFS -- you can just remove them like you would any file in Git. `git-lfs` is smart enough to only download the referenced files when needed to complete a checkout process.

These screenshots were taken manually; they are not generated by the script that you're linking to -- that script is used to update screenshots of the SecureDrop user interface in the documentation.

Reply:
**For future reference**:

> You don't need to do anything special vis-a-vis LFS -- you can just remove them like you would any file in Git. git-lfs is smart enough to only download the referenced files when needed to complete a checkout process.
>
>These screenshots were taken manually; they are not generated by the script that you're linking to -- that script is used to update screenshots of the SecureDrop user interface in the documentation. ([source](https://github.com/freedomofpress/securedrop-docs/pull/141#discussion_r605995840))